.github/workflows/tests.yaml

```
@@ -0,0 +1,236 @@
+          docker buildx bake --file docker-compose.yml \
+            --set *.cache-from=type=gha \
+            --set *.cache-to=type=gha \
+            --set *.output=type=docker   # <-- makes images visible to `docker compose`
+      - name: Start services
+        run: docker compose up --no-build -d
+
+      - name: Wait for services to start
+        run: sleep 30
+
+      - name: Wait for Services to Become Healthy
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:25: Double quote to prevent globbing and word splitting

(shellcheck)